README.md

```diff
@@ -53,6 +53,7 @@ See below for [project information](#project-info).
 * **Rachel Ward** (May 2019), [“When time-based media art in private collections no longer functions, who is responsible for conservation?”](http://ndsr-pma.arlisna.org/2019/05/29/when-time-based-media-art-in-private-collections-no-longer-functions-who-is-responsible-for-conservation/)
 * **Madeline Page Smith** (July 2020), [Caring for the Moving Image in Art Museums: _Matters in Media Art_ and the Stewardship of Time-Based Media Artworks](https://www.nyu.edu/tisch/preservation/program/student_work/2020spring/20s_thesis_Smith_deposit_copy.pdf) (MIAP thesis)
 * **Rene G. Cepeda** (December 2020), [Manual for the Curation and Display of Interactive New Media Art](https://inmamanual.wordpress.com/)
+* **Tate** (April 2021), [Software-based Art Preservation](https://www.tate.org.uk/about-us/projects/software-based-art-preservation)
```

Comment:
Question: The site says ”October 2017 – ongoing”. Is there any reason to write “April 2021”?

Otherwise LGTM.

Reply:
Yeah, Oct 2017 might be the better date, even if the webpage itself is newer!

Thanks for the addition, good find 💜

Reply:
OK, I made the change. I had written April 2021 since it was just announced, and "2017 and ongoing" seemed like it could be outdated. But I don't care either way!

Reply:
Ah right! Don't think I saw that it had been recently announced 😅
Let's go with this for now; if someone from the Tate weighs in, we can update it.

Thanks Ash!